src/main/java/app/attestation/server/AttestationProtocol.java

```diff
@@ -413,6 +414,8 @@ static class DeviceInfo {
                     new DeviceInfo(DEVICE_SM_T510, 3, 4, false /* uses new API */, true, false, OS_STOCK))
             .put("87790149AED63553B768456AAB6DAAD5678CD87BDEB2BF3649467085349C34E0",
                     new DeviceInfo(DEVICE_SM_T835, 1, 2, false, false, false, OS_STOCK))
+            .put("23377439DAD9742BDB722C8D26B8A4CD7E88208DD8827940C1581895AF389630",
```

Comment:
Hi, I would like to add my device too (Samsung A52), how do you get this data (fingerprint?) for a given phone?
Is there any documentation on how to do that?